libs/blocks/global-navigation/global-navigation.js

```diff
@@ -280,6 +280,9 @@ class Gnav {
     if (this.useUniversalNav) {
       delete this.blocks.profile;
       this.blocks.universalNav = toFragment`<div class="feds-utilities"></div>`;
+      this.blocks.universalNav.addEventListener('click', () => {
```

Comment:
```suggestion
      this.blocks.universalNav.addEventListener('click', () => closeAllDropdowns())
```

Reply:
This seems not to work in adobe.com due to the following line (line 289 in utilities.js) in `closeAllDropDowns()`
```javascript
    if ('fedsPreventautoclose' in el.dataset) return;
```

Reply:
IMO emulating a click on the open element is **not** the right path. You should investigate why `fedsPreventautoclose` exists to begin with (might be related to the keyboard navigation using arrow keys, escape, etc.) and otherwise look into adapting the `closeAllDropdowns()` method

Reply:
I don't. The test links don't load the unav because the request for the JS file `403`s. Since it's a one liner, I tested it by adding a breakpoint in the Gnav Constructor and adding in the line manually (you need to do cmd + s for it to work).

Reply:
AFAICT it is related somewhat to the keyboard navigation files. It's difficult to say just from reading the code, but it seems that it makes somewhat of a distinction between specifically a dropdown and the menu as it opens in mobile and tablet viewports.
IMO `closeAllDropdowns` isn't what we're looking for here. What we're looking for is specifically inside the `decorateToggle` method:
```javascript
    const onToggleClick = async () => {
      const isExpanded = toggle.getAttribute('aria-expanded') === 'true';
      toggle.setAttribute('aria-expanded', !isExpanded);
      this.elements.navWrapper.classList.toggle('feds-nav-wrapper--expanded', !isExpanded);
      closeAllDropdowns();
      setCurtainState(!isExpanded);
      toggle.setAttribute('daa-ll', `hamburgermenu|${isExpanded ? 'open' : 'close'}`);

      if (this.blocks?.search?.instance) {
        this.blocks.search.instance.clearSearchForm();
      } else {
        await this.loadSearch();
      }

      if (isExpanded) setHamburgerPadding();
    };

```
The most expedient way to call this function without a fairly significant refactor is by clicking on the element it's attached to. Furthermore it ties the behavior we want to the behavior of the actual html elements the end user interacts with, and not to the abstract way it is done in the code, which is liable to churn. In other words clicking on the toggle element decouples the implementation of the behavior from the implementation of how exactly clicking on the toggle hides/shows the menu.